core/src/main/java/bio/terra/pearl/core/service/site/LocalizedSiteContentService.java

```diff
@@ -52,6 +52,8 @@ public LocalizedSiteContent create(LocalizedSiteContent localSite) {
             dao.setLandingPageId(savedSite.getId(), landingPage.getId());
         }
         savedSite.setLandingPage(landingPage);
+        localSite.getFooterSection().setId(savedSite.getFooterSectionId());
+        savedSite.setFooterSection(localSite.getFooterSection());
```

Comment:
These were not attached to the SiteContent response when a new version was created (so the site footer would disappear from the editor after you saved)